docusaurus/docs/iOS/swiftui/chat-channel-components/overview.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Overview
+---
+
+## What is the Chat Channel?
+
+![Depiction of different message components in the example of the message list.](../../assets/message-components-overview.png)
+
+The chat channel describes all the content that is shown for a specific channel. It consists of three main parts:
+
+- Channel Header
+- Message List
+- Message Composer
+
+There are **many components** that make of the chat channel screen. The **channel header** is shown on the top to give an overview over the channel as well as provide other **helpful information**. It can take care of navigation (e.g. using a back button) and allow the user to see more information about members, name, and other properties of a channel. All of these components are **customizable and easily replaceable**.
```

Comment:
```suggestion
There are **many components** that make up the chat channel screen. The **channel header** is shown on the top to give an overview of the channel as well as provide other **helpful information**. It can take care of navigation (e.g. using a back button) and allow the user to see more information about the name, members, and other properties of a channel. All of these components are **customizable and easily replaceable**.
```